### ISSUE TITLE
100% Crash in 1.0.61 validation layers with RenderDoc

### ISSUE THREAD

**Issue by Erkaman** [baldurk/renderdoc#755]
I tried upgrading lunarg vulkan sdk from version 1.0.57.0 to version 1.0.61.1. But if I then tried running renderdoc on my program, then renderdoc seems to hang on `vkQueuePresentKHR` . Renderdoc worked for  1.0.57.0, but not for this new version. The issue appears both for the stable build v0.91, and for the latest nightly build  `@8a98ba0b`. System specs:

Operating system: Windows 10 Pro, 64-bit
GPU: GeForce GTX 1070
GPU Driver version:	 385.69

Here is my program for reproducing the issue:
https://www.dropbox.com/s/3uzhpfxb850l4r1/vulkan_playground.zip?dl=0

Renderdoc appears to hang at line 1309(`vkQueuePresentKHR`) in the file `main.cpp`, if I launch my program with renderdoc.

All dependencies are included. You can use Cmake to create a visual studio solution. Build and run the project named "vulkan_playground" in the solution. It should look like this:

![capture](https://user-images.githubusercontent.com/602313/30847688-b3b94f7a-a29c-11e7-868e-a727db1d6c3b.PNG)

Please tell me, if you have any issues with compiling and running my program.

**Comment by baldurk** [comment]
I don't get a _hang_ at `vkQueuePresentKHR`, but I certainly get a crash. This seems to be [the 100% crash in the validation layers](https://github.com/KhronosGroup/Vulkan-LoaderAndValidationLayers/issues/2070) I reported, which unfortunately shipped in SDK 1.0.61, which would explain why it started happening when you upgraded.

I built your program in Release mode which your code uses as a switch to flip off the validation layers, and then it seemed to run and capture fine - so I'd suggest just disabling the validation layers while running with renderdoc until the next SDK (or until you build them locally with the fix).
